test/integration/connect/envoy/helpers.bash

```diff
@@ -569,7 +569,7 @@ function docker_consul_for_proxy_bootstrap {
   local DC=$1
   shift 1
 
-  docker run -i --rm --network container:envoy_consul-${DC}_1 consul-dev "$@"
+  docker run -i --rm --network container:envoy_consul-${DC}_1 consul-dev "$@" 2> /dev/null
```

Comment:
This isn't needed specifically for this change but when I was trying to use arm images I noticed that any docker run errors/warnings are captured along with the `-bootstrap` output which malforms the json.